crates/shared/src/price_estimation/instrumented.rs

```diff
@@ -67,6 +101,14 @@ struct Metrics {
     /// price estimation times
     #[metric(labels("estimator_type"))]
     price_estimation_times: HistogramVec,
+
+    /// number of native price requests sent to each estimator
```

Comment:
Probably good to add a caveat here. Most native estimators are actually regular estimators with a wrapper. In those cases native price requests will get counted as regular `price_estimates`. That means that `native_price_estimates` only counts requests for exclusively native estimators.

Reply:
Thought about this a bit more and I think we can straight up reuse the 2 existing metrics. Ultimately it doesn't make a huge difference whether an estimator is strictly a native price estimator under the hood or a regular estimator.